crunchy.ts

```diff
@@ -2166,7 +2166,7 @@ export default class Crunchy implements ServiceClass {
       merger.cleanUp();
   }
 
-  public async listSeriesID(id: string): Promise<{ list: Episode[], data: Record<string, {
+  public async listSeriesID(id: string, data: CrunchyMultiDownload | undefined = undefined): Promise<{ list: Episode[], data: Record<string, {

```

Comment:
Is there a reason to use `data: CrunchyMultiDownload | undefined = undefined` here instead of `data?: CrunchyMultiDownload`?

Reply:
changed, my vscode auto-suggests things so i don't usually argue with what it suggests.